src/main/java/webserver/servlet/Controller.java

```diff
@@ -0,0 +1,22 @@
+package webserver.servlet;
+
+import webserver.enums.HttpMethod;
+import webserver.request.HttpRequest;
+import webserver.response.HttpResponse;
+
+public interface Controller {
+
+    default void service(HttpRequest httpRequest, HttpResponse httpResponse) {
+        if (httpRequest.getMethod() == HttpMethod.POST) {
+            doPost(httpRequest, httpResponse);
+            return;
+        }
+
+        doGet(httpRequest, httpResponse);
+    }
+
+    void doPost(HttpRequest httpRequest, HttpResponse httpResponse);
+
+    void doGet(HttpRequest httpRequest, HttpResponse httpResponse);
```

Comment:
default 구현을 제공하는것이 어떨까요 ?

```suggestion
    default void doPost(HttpRequest httpRequest, HttpResponse httpResponse) {
       // throw Exception 
    }

    default void doGet(HttpRequest httpRequest, HttpResponse httpResponse) {
       // throw Exception 
    }
```

Reply:
옙옙 다같이 디폴트로 만들까하다가 말았었는데 수정해보겠습니다!!